pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -744,4 +744,18 @@ BEGIN
         COMMIT;
     END LOOP;
 END;
-$$;
\ No newline at end of file
+$$;
+
+CREATE OR REPLACE FUNCTION SCHEMA_PROM.is_stale_marker(value double precision)
```

Comment:
This would be great to document for users as well, and is great UX. The bit twiddling could be non trivial for some users.

Reply:
yeah will document everything in schema_prom